CONTRIBUTING.md

```diff
@@ -64,33 +14,12 @@ After doing that open a merge/pull request.
 
 If your fix/feature is accepted it will be merged into the develop-branch.
 
-#################
-TODO: 
-- How to make branches out of issues
-- How to work with branches
-- Board usage
-- Gitlab in general: features
-- Wiki vs Web
-#######################
-
-### FAQ
-
 ## Reporting bugs and requesting features
 If you got a question or a problem and need support from our team feel free to contact us.
 You can do this via [email](mailto:dev@jupedsim.org).
```

Comment:
I would remove this mail and keep things in GitHub's issue tracker.

Reply:
changed.